jwt/algorithms.py

```diff
@@ -171,6 +199,7 @@ def sign(self, msg, key):
     def verify(self, msg, key, sig):
         return constant_time_compare(sig, self.sign(msg, key))
 
+
```

Comment:
This (and the other extra space) was to fix flake8 errors reported by the tests. It seemed to be failing to pass without the changes (but maybe that was just a local configuration on my box)